.pipelines/e2e-job.yaml

```diff
@@ -18,7 +18,7 @@ variables:
   image.dir: $(Build.ArtifactStagingDirectory)
   image.artifact.name: drop
   KUBECONFIG: $(System.DefaultWorkingDirectory)/kubeconfig.json
-  image.tag: 0.1.0
+  checkout.tag: 0.1.0
```

Comment:
Ah I wasn't clear here. It's okay that this was `image.tag`. I just also wanted you to define `checkout.tag: $(image.tag)-${UNIQUE_TAG}` (or something like that with proper syntax) and then pass that value in as an environment variable to the rest of the scripts.

Reply:
going to make a global var called "CHECKOUT_TAG" in the package.helm-chart.yaml, right after generating the unique tag